swagger/actokids.yaml

```diff
@@ -57,10 +57,10 @@ paths:
               $ref: '#/definitions/Event'
         '500':
           description: Failure to connect or save to the database
-  /event/{eventID}:
+  '/events/{eventID}':
     get:
       tags:
-        - event
+        - Events
```

Comment:
I think eventID format should be "string", since the uuid is a hexadecimal number, and I don't think that's an option as a Swagger data type

Reply:
I think I got all of them changed to string